src/GraphQLNormalizr.js

```diff
@@ -20,7 +20,7 @@ import {
 } from './utils'
 import { CACHE_READ_ERROR, CACHE_WRITE_ERROR, PAGEINFO_WITH_USE_CONNECTIONS_FALSE, } from './constants'
 
-const isProd = process?.env?.NODE_ENV === 'production'
+const isProd = true
```

Comment:
I reviewed every instance this is used and agree it's a fine change; if we can do without though, again I think we'd have a better change of merging to the main fork.

Reply:
I attempted adjusting the rollup config however it lead to the same error when installed onto the main (`process is not defined`).

https://github.com/chronograph-pe/graphql-normalizr/pull/3/commits/48e428461738ce242e3f16e83cf79781eb0221cc

To note, I'm not optimistic this will get accepted into the main repo as it is very specific to `postgraphile` and goes against the GQL spec. _Maybe_ the `postgraphile` engineers themselves will accept it as a sub-package for their repository.

Feel free to take a stab at it, but we can also just make a separate branch that excludes this line and open that against the main repo.

Separately, I've reverted the package-lock.json changes.